# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1323068542 18000
#      Mon Dec 05 02:02:22 2011 -0500
# Branch stable
# Node ID 95bcbecf5801230c804fe26e55eb4ef71b063082
# Parent  5289d7c2460db8c55912320ce8a0f56a9e44c9bc
remove empty ARPACK files

* arpack/src/dnaupe.f, arpack/src/snaupe.f: Delete empty files.
* libcruft/arpack/module.mk (libcruft_la_SOURCES): Delete them from
the list.

diff --git a/libcruft/arpack/module.mk b/libcruft/arpack/module.mk
--- a/libcruft/arpack/module.mk
+++ b/libcruft/arpack/module.mk
@@ -24,17 +24,16 @@ libcruft_la_SOURCES += \
   arpack/src/csortc.f \
   arpack/src/cstatn.f \
   arpack/src/dgetv0.f \
   arpack/src/dlaqrb.f \
   arpack/src/dnaitr.f \
   arpack/src/dnapps.f \
   arpack/src/dnaup2.f \
   arpack/src/dnaupd.f \
-  arpack/src/dnaupe.f \
   arpack/src/dnconv.f \
   arpack/src/dneigh.f \
   arpack/src/dneupd.f \
   arpack/src/dngets.f \
   arpack/src/dsaitr.f \
   arpack/src/dsapps.f \
   arpack/src/dsaup2.f \
   arpack/src/dsaupd.f \
@@ -49,17 +48,16 @@ libcruft_la_SOURCES += \
   arpack/src/dstats.f \
   arpack/src/dstqrb.f \
   arpack/src/sgetv0.f \
   arpack/src/slaqrb.f \
   arpack/src/snaitr.f \
   arpack/src/snapps.f \
   arpack/src/snaup2.f \
   arpack/src/snaupd.f \
-  arpack/src/snaupe.f \
   arpack/src/snconv.f \
   arpack/src/sneigh.f \
   arpack/src/sneupd.f \
   arpack/src/sngets.f \
   arpack/src/ssaitr.f \
   arpack/src/ssapps.f \
   arpack/src/ssaup2.f \
   arpack/src/ssaupd.f \
diff --git a/libcruft/arpack/src/dnaupe.f b/libcruft/arpack/src/dnaupe.f
deleted file mode 100644
diff --git a/libcruft/arpack/src/snaupe.f b/libcruft/arpack/src/snaupe.f
deleted file mode 100644
